README.md

```diff
@@ -27,16 +28,18 @@ What is the purpose of the `alt` attribute? e.g:
 <img src='https://jesse.sh/img/me.jpg' alt='Jesse Shawl'/>
 ```
 
-```text
-Your answer here.
+```
+2. the "alt" attribute is used to describe photos in metadata, used primarily to describe photos on web pages to the blind.
+
 ```
 
 ### Question #3
 
 What is the purpose of the `<head></head>` tag in HTML?
 
-```text
-Your answer here.
+```
+3. the <head> tag describes the section of the web page the user cannot see e.g. behind the scenes operation.
+
 ```
```

Comment:
:+1:  used to contain any "rules", meta data, and other information that affects the webpage but does not directly show up on the webpage itself.